deploy/kustomize/driver/pmem-csi.yaml

```diff
@@ -147,10 +147,6 @@ spec:
         app.kubernetes.io/instance: pmem-csi.intel.com
         pmem-csi.intel.com/webhook: ignore
     spec:
-      securityContext:
-        runAsNonRoot: true
```

Comment:
Does running the controller as non-root(though not choosing the UID) is still not relevant?

Reply:
Have you seen the commit message? There I am trying to discuss that aspect.

In a nutshell, actual users will still run this as non-root.

Reply:
docs/design.md also gets updated to explain this.

Reply:
My point is can't we keep `runAsNotRoot: true` by not setting the UID?

Reply:
But on upstream Kubernetes we do run as root. We can't keep `runAsNotRoot: true` unless we set the fixed UID, can we?